test-poc/framework/src/main/java/org/keycloak/test/framework/server/DistributionKeycloakTestServer.java

```diff
@@ -1,15 +1,46 @@
 package org.keycloak.test.framework.server;
 
+import org.keycloak.it.utils.RawKeycloakDistribution;
+
+import java.util.LinkedList;
+import java.util.List;
+
 public class DistributionKeycloakTestServer implements KeycloakTestServer {
 
+    private boolean debug = false;
+    private boolean manualStop = true;
+    private boolean enableTls = false;
+    private boolean reCreate = false;
+    private boolean removeBuildOptionsAfterBuild = false;
+    private int requestPort = 8080;
+    private RawKeycloakDistribution keycloak;
+
     @Override
     public void start(KeycloakTestServerConfig serverConfig) {
-        throw new RuntimeException("Method not implemented!");
+        keycloak = new RawKeycloakDistribution(debug, manualStop, enableTls, reCreate, removeBuildOptionsAfterBuild, requestPort);
+
+        // Set environment variables user and password for Keycloak Admin used by Keycloak instance.
+        keycloak.setEnvVar("KC_BOOTSTRAP_ADMIN_USERNAME", serverConfig.adminUserName().get());
+        keycloak.setEnvVar("KC_BOOTSTRAP_ADMIN_PASSWORD", serverConfig.adminUserPassword().get());
+
+        List<String> rawOptions = new LinkedList<>();
+        rawOptions.add("start-dev");
```

Comment:
Do we want to add options from `org.keycloak.test.framework.server.KeycloakTestServerConfig#options` now?

Reply:
@pedroigor It is copied from the EmbeddedServer setup, so not necessary to have it right now, but will be used in the future.